pyproject.toml

```diff
@@ -1,5 +1,4 @@
 [tool.towncrier]
-package = "pulpcore.cli.ostree"
```

Comment:
This allows running towncrier without installing the actual package as long as providing the correct version, right?

Reply:
I suppose yes? I am trying to perform a release and this seemed to be the only discrepancy between pulp-cli-gem (successfully released 3 weeks ago) and pulp-cli-ostree.

I am seeing some errors in the release pipeline and I thought this could resolve the issues: https://github.com/pulp/pulp-cli-ostree/actions/runs/6891454589/job/18746543474#step:6:81.

Reply:
It looks right to me.

Reply:
It needs to provide the proper version anyway, because it runs just before bump2version is doing it's magic.
https://github.com/pulp/pulp-cli-ostree/blob/main/.ci/scripts/release.sh#L22

Reply:
The release was successful thanks to this change.